lib/auth/webauthncli/fido2.go

```diff
@@ -585,6 +595,20 @@ func runOnFIDO2Devices(
 	return trace.Wrap(err)
 }
 
+func doEagerPINPrompt(passwordless bool, devices []deviceWithInfo) bool {
```

Comment:
Getting really deep into nitpick territory here, but `doEagerPINPrompt` reads to me like this function has side effects, maybe something like `shouldDoEagerPINPrompt` might convey the intent more clearly? Up to you regardless.

Reply:
I agree, didn't like the other name tbh. Done.